pages/account/index.tsx

```diff
@@ -32,7 +56,16 @@ export async function getServerSideProps({ req }) {
     }
   }
 
-  return {
-    props: { isAuthenticated },
+  const id = patronTokenResponse.decodedPatron.sub
+  let props
+  try {
+    const { checkouts, holds, patron, fines } = await MyAccountFactory(id)
+    props = { checkouts, holds, patron, fines, isAuthenticated }
```

Comment:
This can be returned directly without assigning to an intermediate 'props' var.

Reply:
True. I suggested this approach because then this function only has one return and it's at the end. It's a preference thing but whatever works best.

Reply:
I'm going to revert it. We have a return statement higher up anyways, and I personally think it feels more explicit to have the return statement split by try/catch